### ISSUE TITLE
banned-domains: Add UI for managing domain banlist + option for custom ban reason

### ISSUE THREAD

**Issue by ajhalili2006** [thedevs-network/kutt#469]
I would like to suggest adding a UI for managing the domain banlist (amending ban reason, lifting user-level and domain-level bans) and also have a way to customize ban reason. The ban reason `malware or scam` seems to be too much wide.

**Comment by poeti8** [comment]
Yes we have a plan on adding this UI. I'm gonna work on it after https://github.com/thedevs-network/kutt/pull/440 gets merged.
